barna-ajedrez/src/components/TournamentComponents/formToRender.js

```diff
@@ -0,0 +1,225 @@
+import React, { Component } from 'react';
+import { noVisibility, updateTournamentList } from '../../actions/actions';
+import { connect } from 'react-redux';
+import axios from'axios';
+
+class FormToRender extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      nameOfTournament: '',
+      clubCreated: false,
+      location: '',
+      clubPhone: '',
+      clubPhone2: '',
+      clubEmail: '',
+      clubEmail2: '',
+      startDate: '',
+      additionalInfo: '',
+    };
+  }
+  hideForm = () => {
+    this.props.turnOffVisibility();
+  }
+  handleChange = e => {
+    console.log(e.target.value);
+
+    let value = e.target.value;
+    let name = e.target.name;
+
+    this.setState({ [name]: value });
+  };
+
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const value = this.state;
+    // const eChecker = this.state.clubEmail === this.state.clubEmail2 ? true : false;
+    // const nChecker = this.state.clubNumber === this.state.clubNumber2 ? true : false;
+    // // const
+
+    // if (!eChecker) {
+    //   alert('please make sure that your emails are the same');
+    // } else if (!nChecker) {
+    //   alert('please make sure that the phone numbers are the same');
+    // } else {
+      console.log('STATE BEING SENT', value);
+
+      axios
+        .post('http://localhost:3001/CreateTournament', {
+          nameOfTournament: value.nameOfTournament,
+          clubCreated: value.clubCreated,
+          location: value.location,
+          clubPhone: value.clubPhone,
+          clubEmail: value.clubEmail,
+          startDate: value.startDate,
+          additionalInfo: value.additionalInfo
+        })
+        .then(res => {
+          console.log(res);
+          console.log('Tournament Created Succesfully');
+        })
+        .catch(err => {
+          // properly receives
+          console.log('this is the error', err);
+          alert('Was not');
+        });
+      //
+      axios
```

Comment:
Make the API to return the data of the newly created tournament on the previous POST request instead.
Also doing it this way you're not waiting until the previous request succeeded.